cmd/gardenlet/app/app.go

```diff
@@ -344,9 +339,14 @@ func (g *garden) Start(ctx context.Context) error {
 				return nil, err
 			}
 
+			seedNamespace := gardenerutils.ComputeGardenNamespace(g.config.SeedConfig.SeedTemplate.Name)
```

Comment:
Instead of computing this again, maybe make it from L291 reusable?

Reply:
Addressed in [2b7d232](https://github.com/gardener/gardener/pull/8447/commits/2b7d232a6e327d19511f635ce94840ac44e0faf3)